### PR TITLE
FIxed conflict and improved some background staff

### PR DESCRIPTION
**Pull request by nabilwardeh**

### CODE REVIEW

File: api/src/controllers/FeedController.js [hunk 1]
```diff
@@ -31,7 +30,6 @@ class FeedController extends Controller {
         resolved: node.post.resolved || false,
         createdAt: node.rel.properties.at,
         createdAtSince: moment(node.rel.properties.at).fromNow(),
-        commentCount: node.commentCount,
```
Comment (kiyaGu) [inline api/src/controllers/FeedController.js]: Why did you remove the commentCount? we use it in displaying the comments or did you replace it with an alternative?

Comment (nabilwardeh) [reply api/src/controllers/FeedController.js]: This is to a temporary fix for grouping. And no the comment count comes from the data when comments are retrieved in the comment component

Comment (nabilwardeh) [reply api/src/controllers/FeedController.js]: This is to a temporary fix for grouping. And no the comment count comes from the data when comments are retrieved in the comment component

**Comment by kiyaGu** [comment]
approved Nabil's PR
